src/openapiSpec.ts

```diff
@@ -44,6 +43,12 @@ const globalName = (global: SanitizedGlobalConfig): string => {
 
 type ComponentType = 'schemas' | 'responses' | 'requestBodies'
 
+const baseQueryParams: Array<OpenAPIV3.ParameterObject & OpenAPIV3_1.ParameterObject> = [
+  { in: 'query', name: 'depth', schema: { type: 'number' } },
+  { in: 'query', name: 'locale', schema: { type: 'string' } },
+  { in: 'query', name: 'fallback-locale', schema: { type: 'string' } },
```

Comment:
I believe that available locales could (and eventually should) be extracted from the Payload config. Then, we could expose them as `{type: 'string', enum: ['en', ...]}`. If you just need a quick patch now, please make an issue to track this. Or just a code TODO.

Reply:
Good idea. I have a local setup for my project right now, so there's no hurry to release on NPM.